src/AppInstallerRepositoryCore/Rest/Schema/1_1/Json/ManifestDeserializer_1_1.cpp

```diff
@@ -139,7 +139,7 @@ namespace AppInstaller::Repository::Rest::Schema::V1_1::Json
                     // Only add when it is valid
                     if (installerReturnCode != 0 && returnResponse != ExpectedReturnCodeEnum::Unknown)
                     {
-                        if (!installer.ExpectedReturnCodes.insert({ installerReturnCode, returnResponse }).second)
+                        if (!installer.ExpectedReturnCodes.insert({ installerReturnCode, AppInstaller::Manifest::ManifestInstaller::ExpectedReturnCodeInfo{returnResponse} }).second)

```

Comment:
>{returnResponse} [](http://example.com/codeflow?start=154&length=16)

maybe { InstallationNotes, "" }to be clear that v1.1 does not have response string

Reply:
updated to suggestion